python-lib/ontology_tagger.py

```diff
@@ -205,17 +201,21 @@ def tag_and_format(
         -Split sentences by applying sentencizer on documents
```

Comment:
Nitpick: that method is quite long, it would be nice to break it down in sub-methods and then call these sub-methods from this one

Reply:
baed249

Reply:
Getting slightly better. I left a few comments on this commit for you to refine the names of the new methods... And don't forget docstrings!

Reply:
f5c2a8b